dissect/target/plugins/os/windows/adpolicy.py

```diff
@@ -62,6 +63,29 @@ def check_compatible(self):
         if len([d for d in self.dirs if d.exists()]) <= 0:
             raise UnsupportedPluginError("No AD policy directories found")
 
+    def _xmltasks(self, policy_dir):
+        for task_file in policy_dir.rglob("ScheduledTasks.xml"):
+            try:
+                task_file_stat = task_file.stat()
+                xml = task_file.read_text()
+                tree = ElementTree.fromstring(xml)
+                for task in tree.findall("Task"):
+                    properties = task.find("Properties")
+                    yield ADPolicyRecord(
+                        last_modification_time=task_file_stat.st_mtime,
+                        last_access_time=task_file_stat.st_atime,
+                        creation_time=task_file_stat.st_ctime,
+                        guid=task.attrib.get("uid"),
+                        key="XML",
+                        value=properties.attrib.get("appName"),
+                        size=len(xml),
```

Comment:
Shouldn't the size then not also be the size of the task data below?

Reply:
Good point. Oversight.